global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+    "version": "3.1.400",
+    "rollForward": "latestFeature"
```

Comment:
Wouldn't latestfeature pin it to 3.1.xxx and latestPatch to 3.1.4xx? 
(based on [this article](https://andrewlock.net/exploring-the-new-rollforward-and-allowprerelease-settings-in-global-json/#understanding-the-various-rollforward-policies))

Reply:
That's correct, but here we are setting 3.1.400 as the minimum baseline. So we pin to 3.1._N_ where _N_ >= 400.